.github/workflows/chromatic.yml

```diff
@@ -32,19 +32,11 @@ jobs:
           yarn tokens build
           yarn tailwind-preset build
 
-      - if: ${{ github.event.inputs.force != true && github.ref != 'refs/heads/master' }}
-        name: "Chromatic dry run"
-        run: echo "Publishing dry run to Chromatic"
-
-      - if: ${{ github.event.inputs.force == true || github.ref == 'refs/heads/master' }}
-        name: "Chromatic publish"
-        run: echo "Would publish to Chromatic"
-
-      # - name: Publish to Chromatic
-      #   uses: chromaui/action@v1
-      #   with:
-      #     projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-      #     workingDir: packages/orbit-components
-      #     buildScriptName: build:storybook
-      #     exitZeroOnChanges: false
-      #     dryRun: ${{ github.event.inputs.force != true && github.ref != 'refs/heads/master' }}
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          workingDir: packages/orbit-components
+          buildScriptName: build:storybook
+          exitZeroOnChanges: false
+          dryRun: ${{ github.event.inputs.force != 'true' && github.ref != 'refs/heads/master' }}
```

Comment:
Oh, it needs correction based on what we discussed today. Well spotted. The `dryRun` will be set to true on every PR / push so that we test that chromatic would be able to deploy but without actually capturing any snapshot or deploying anything.
If `force` is true (configurable on manual deploy), it will not execute with `dryRun` and, therefore, deploy chromatic and perform tests.
Updated

Reply:
ok thank 👍